pkg/ttnpb/field_mask_validation.go

```diff
@@ -141,10 +141,9 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 		"ids.device_id",
 		"ids.join_eui",
 		"session.dev_addr",
-		"session.keys.app_s_key",
```

Comment:
Breaks API

Reply:
It does not break any promises in https://github.com/TheThingsNetwork/lorawan-stack#commitments-and-releases as far as I am concerned

Reply:
It may break console, in case they use `session.*_key` directly, in which case we need to fix that and bump minor

Reply:
Yes it does, because in the past we allowed `session.keys.app_s_key` and now we don't anymore. That simply breaks clients.

Reply:
In https://github.com/TheThingsNetwork/lorawan-stack/pull/1550#issuecomment-550311221 you explicitly pointed out point 3 of "commitments and releases", which reads "We will not break the public command-line interface and configuration within the major version. This means that you can safely build scripts and migrate configurations".
This PR is compliant with this and 3 other commitments.

So do you propose to allow setting `*_key.{encrypted_key,kek_label}` fields? That is the only way to not break clients, who use the registry API directly.